broker/lib/fabrik/DirectorInstance.js

```diff
@@ -231,6 +302,23 @@ class DirectorInstance extends BaseInstance {
       );
   }
 
+  delete20(params) {
```

Comment:
what is the diff between delete20 and delete10

Reply:
We are [verifying](https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/304/files#diff-46e37f17222369258c2e785ea9671484R293) `deploymentLockStatus` in delete10